experimental/packages/opentelemetry-sdk-metrics-base/test/Instruments.test.ts

```diff
@@ -199,7 +199,7 @@ describe('Instruments', () => {
           type: InstrumentType.UP_DOWN_COUNTER,
           valueType: ValueType.INT,
         },
-        dataPointType: DataPointType.SINGULAR,
```

Comment:
Would you mind adding assertions on `isMonotonic` for up down counters?

Reply:
Oh, I completely missed the assertions here. Added in [0291a65](https://github.com/open-telemetry/opentelemetry-js/pull/3079/commits/0291a654eaac263421c5b221c26742c9c7ff2b04). Thanks for pointing this out! :slightly_smiling_face: